doc/classes/AnimationPlayer.xml

```diff
@@ -237,6 +237,7 @@
 		</member>
 		<member name="current_animation" type="String" setter="set_current_animation" getter="get_current_animation" default="&quot;&quot;">
 			The name of the current animation, "" if not playing anything. When being set, does not restart the animation. See also [method play].
```

Comment:
I know it's not part of your PR but this docstring could be expressed more clearly:

```suggestion
			The name of the currently playing animation. If no animation is playing, the property's value is an empty string. Changing this value does not restart the animation. See [method play] for more information on playing animations.
```

Reply:
The animation tracks should be mentioned somehow, because they use this internally (similar to audio tracks and `playing` property). Also you can pin AnimationPlayer and create a keyframe for `current_animation` in another AnimationPlayer, which will create an animation track.
(by animation track I mean the option at the bottom
![image](https://user-images.githubusercontent.com/2223172/91619118-7d979200-e98c-11ea-8d02-6c0d0e30ebfe.png)
)

Also it's not really read-only, you can set it in script.

Reply:
Alright, so I think I don't understand the exact purpose. The property itself isn't serialized in scenes, but it is exposed so you can use it with animation tracks? What would you use it for in code? I've personally never found a use for this one as there are more efficient methods in the class.

The current docstring makes it sound like you're not supposed to assign values to this property directly.

Reply:
Yeah, the documentation is a bit misleading. Assigning new `current_animation` will play the animation, but the animation won't be reset when you assign the same one (this mention could be removed if it's expected). It was kind of useful back when `play()` did reset animation (it doesn't anymore), not sure if it has any use in scripts now.

The main use for the `current_animation` are "animation tracks" (mostly internally) and getting the current animation. But assigning/animating it is possible too.